app/device/test.py

```diff
@@ -0,0 +1,7 @@
+import altimeter
```

Comment:
test file should go in `/test/device/test_altimeter.py`
and look something like [this](https://github.com/uOstar/barista/blob/master/test/device/test_gps.py)

this uses pytest and should mock the behaviour of the actual sensor